bitnami/chainloop/Chart.yaml

```diff
@@ -14,7 +14,7 @@ annotations:
     - name: dex
       image: docker.io/bitnami/dex:2.41.1-debian-12-r0
 apiVersion: v2
-appVersion: 0.95.4
+appVersion: 0.95.5
```

Comment:
are we in charge of bumping this or is it Bitnami's automation?

Reply:
The appVersion is done automatically. The `version` section of the Chart.yaml has to be done automatically.